src/envs/react/theme-with-centering.tsx

```diff
@@ -1,6 +1,6 @@
 export {};
 
-// TODO - 
+// TODO -
```

Comment:
I saved this file because it's not saved correctly and cause the prettier error in the github action.
I can remove this, but then I will get a prettier error in github action.

Reply:
oh damn. ok.

Reply:
I'll tag all the components.